src/nylas-connection.js

```diff
@@ -142,13 +142,18 @@ module.exports = class NylasConnection {
         if (options.json === false) {
           body = JSON.parse(body);
         }
+        if (typeof error === 'string') {
```

Comment:
Is this necessary? Under what circumstances would the incoming error be a string, do you have an example?

Reply:
I'm sorry @Gerdie, I don't quite remember why I did this.

Reply:
Let's remove it for now, and we can revisit it if we get another bug report? Otherwise, lgtm!